api/pages/users_page.py

```diff
@@ -1,34 +1,34 @@
-"""This file contains the class Employee page with the methods: create_employee_record,
-get_employee, get_all_employee and delete_employee"""
+"""This file contains the class Users page with the methods: create_user_record,
+get_user, get_all_users and delete_user"""
 
 from api.pages.base_page import BasePage
 from api.utils.api_methods import ApiMethod
 from api.utils.api_body import api_body
 
 
-class EmployeePage(BasePage):
-    """Class representing employee page."""
-    def create_employee_record(self):
+class UsersPage(BasePage):
+    """Class representing users page."""
+    def create_user_record(self):
         """Method returning the response with status OK or with too many requests
          based on url, api method post and api body"""
-        url = "https://dummy.restapiexample.com/api/v1/create"
+        url = "https://reqres.in/api/users"
         return self.make_api_request(url, api_method=ApiMethod.POST.value,
                                      api_body=api_body)
 
-    def get_employee(self, employee_id):
+    def get_user(self, user_id):
         """Method returning the response with status OK or with too many requests
          based on url and api method get"""
-        url = f"https://dummy.restapiexample.com/api/v1/employee/{employee_id}"
+        url = f"https://reqres.in/api/users/{user_id}"
         return self.make_api_request(url, api_method=ApiMethod.GET.value)
 
-    def get_all_employees(self):
+    def get_all_users(self):
         """Method returning the response with status OK or with too many requests
          based on url and api method get"""
-        url = "https://dummy.restapiexample.com/api/v1/employees"
+        url = "https://reqres.in/api/users?page=2"
```

Comment:
de ge pui users?page=2 , not oki sa folosim query param in metoda,fa cumva sa ai o metoda sa primeasca asta